### ISSUE TITLE
count deleted accepts/declines/rejects

### ISSUE THREAD

**Issue by NovemLinguae** [APerson241/afchistory#12]
I thought of a way to do this without being an admin. Accepts, declines, and rejects leave a user talk message, and user talk pages are almost never deleted. We can query the `revision` table and search for `comments` containing AFCH accept/decline/reject text.

would want to search the user talk namespace, since user talk pages on enwiki are almost never deleted, per policy.

Example: https://quarry.wmcloud.org/query/74453

Could do this for comments too, but wouldn't catch them all, since posting to user talks notifying about comments is a check box that is sometimes unchecked.

**Comment by enterprisey** [comment]
just confirming, the idea here is to catch when a draft gets declined, and then deleted, and then recreated and submitted?

**Comment by NovemLinguae** [comment]
Not exactly. My idea is to make the algorithms used to generate these counts more accurate:

<img width="180" alt="image" src="https://github.com/enterprisey/afchistory/assets/79697282/23c5eda3-ce6e-4873-a800-1b181aceb3ca">

Currently these do not check deleted pages. Using my proposed algorithm, it would be possible to get these to include deleted pages without being an admin. Could also do it with one replica database query instead of JavaScript and multiple API queries.

We'd leave the code that generates the list of drafts alone. That can continue using JavaScript and multiple API queries.

<img width="493" alt="image" src="https://github.com/enterprisey/afchistory/assets/79697282/52ae1371-e54f-45cd-81e5-c03e90bd7c51">
